<launch>
  <include file="$(find rise_helios2)/launch/dip_stream.launch">
    <arg name="device_user_id" default="cam0" />
    <arg name="kinect_sensor_sn" default="" />   <!-- Put serial number if you use multiple kinect -->
    <arg name="color_resolution" default="720P"/> <!--options: 720P, 1080P, 1440P, 1536P, 2160P, 3072P -->
    <arg name="use_kinect"   default="true"/>
    <arg name="helios_fps"   default="15"/>
    <arg name="kinect_fps"   default="15"/>
  </include>

  <include file="$(find ros_unet)/launch/ex.launch">
    <arg name="rgb"   default="/cam0/helios2/rgb_rect"/>
    <arg name="depth" default="/cam0/helios2/depth_rect"/>
    <arg name="info"  default="/cam0/helios2/camera_info_rect"/>
  </include>

</launch>
